awesomerc.lua

```diff
@@ -562,13 +534,4 @@ client.connect_signal("request::titlebars", function(c)
         layout = wibox.layout.align.horizontal
     }
 end)
-
--- Enable sloppy focus, so that focus follows mouse.
```

Comment:
and how to disable it now?

Reply:
The permission system or by disconnecting the request. This is how all other `request::` are already used. The `request::` API was originally designed to move some hard coded behavior into something you could disable and replace by your own logic. In this case, it was used to move things that boilerplate for most people (like setting the border) into something invisible, but easy to disable.

Add to that the new permission API, and you can now define what you wish to enable/disable in a per-client basis. Another pull request (because the `ruled.client` one is already maxed-out) will add an easy way to do that from the client rule. The end result will be superior and easier to manipulate than having the code in `rc.lua`. See the new "Permission" section in the doc (you need to click "Show more" to see them, but they are in the summary).

Reply:
but how you going to disconnect lambda?

Reply:
ping, that's still a noticeable functional regression in the current master

Reply:
Can you provide an example based on the default config where the behavior changed?

Reply:
it's not possible anymore to disable a "sloppy focus", in order to disconnect a signal it should be an exported function of the module, not an anonymous or local one

Reply:
It is still possible using the permission API, but fair enough. I worked on an API level system last weekend, but it isn't ready for review yet. These API changes are "required" going forward for the next round. You will like them because they get us closer to HiDPI.

But right, the autofocus stuff needs to go back to `rc.lua` for API level 4, there is no way to detect why the user removed it (either because he/she use the new API or because they don't want it)

Reply:
so to simplify the things up for now -- what about just moving the anonymous function to be a module-level method?

Reply:
(in a similar manner like `client.disconnect_signal("manage", awful.rules.apply)`)